### DISCUSSION TITLE
Contributing guidelines still mention Poetry instead of PDM

### DISCUSSION THREAD

**Discussion by BartSchuurmans** [fastapi/sqlmodel#998] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

The [contributing guidelines](https://sqlmodel.tiangolo.com/contributing/) mention Poetry for development of SQLModel, but I see the project has migrated to PDM

### Operating System

Linux

### SQLModel Version

n/a

### Python Version

n/a

**Comment by YuriiMotov** [comment]
No Poetry anymore..
